resources/configurationFiles/analysisCopyNumberEstimation.xml

```diff
@@ -121,10 +121,10 @@
         <cvalue name='SNP_SUFFIX' value='snp.tab.gz' type="string"/>
         <cvalue name='snp_min_coverage' value='5' type="integer"/>
 
-        <cvalue name='MALE_FAKE_CONTROL_PRE' value="TODO /icgc/ngs_share/assemblies/hg19_GRCh37_1000genomes/databases/LocalControls/ACEseq_GenericMaleControl/GenericMaleControl.chr" type='path'
+        <!--cvalue name='MALE_FAKE_CONTROL_PRE' value="TODO /icgc/ngs_share/assemblies/hg19_GRCh37_1000genomes/databases/LocalControls/ACEseq_GenericMaleControl/GenericMaleControl.chr" type='path'
```

Comment:
Why is this quoted? Is it not required anymore at all? Does it mess things up? Can this be completely deleted?

Reply:
these files are needed for noControl cases. we would have to define hg38 samples to be used as male/female genericControl samples. We still will need the *FAKE_CONTROL_PRE cvalues.

Reply:
This was a TODO from Tom's version. Yes, it need only in no-control cases, and normal workflow runs through without this variable. @GWarsow does a leftover to hg38 coordinates works here?